File: SEM3/WTLAB/Lab7/Q5.java

class OverloadArea {
    double area(double r) {
        if (r < 0) {
            return -1;
        }
        return Math.PI * (r * r);
    }

    double area(double l, double b) {
        if (l < 0 || b < 0) {
            return -1;
        }
        double areaOfRectangle = l * b;
        return areaOfRectangle;
    }

    double area(int a, double b, double c) {
        double temp = (a + b + c);
        double s = temp / 2;
        double areaOfTriangle = Math.sqrt(s * (s - a) * (s - b) * (s - c));
        return areaOfTriangle;
    }
}

public class Q5 {
    public static void main(String[] args) {
        OverloadArea a = new OverloadArea();
        System.out.println("Area of Circle-> " + a.area(15));
        System.out.println("Area of Rectangle-> " + a.area(15, 30));
        System.out.println("Area of Triangle-> " + a.area(2, 4, 3));
    }
}

File: SEM3/oop lab/Lab 4/q5.cpp
#include<iostream>
using namespace std;
class areas
{

public:
int area(int l=5, int b=1)
{
    return(l*b);
}
float area1(float r=3)
{
    return(3.14*r*r);
}
float area2(float bs=2,float ht=1)
{
   return((bs*ht)/2);
}
};


int main()
{
    areas a;
   
    cout<<"\nArea of rectangle is "<<a.area();
    cout<<"\nArea of circle is "<<a.area1();
    cout<<"\nArea of triangle is "<<a.area2();
return 0;
}

